core/domain/platform_parameter_domain.py

```diff
@@ -781,7 +784,7 @@ def _validate_feature_flag(self):
                             'production environment.')
 
     @classmethod
-    def from_dict(cls, param_dict):
+    def from_dict(cls, param_dict: Dict[str, Any]):
```

Comment:
Why `Any`?

Reply:
because fields of PlatformParameter object contain (str, data_types, int, bool)

Reply:
ditto as above

Reply:
done!